### DISCUSSION TITLE
watchdog-keepalive in the RDEPENDS:${PN}

### DISCUSSION THREAD

**Discussion by bchoineubility** [OE4T/meta-tegra#1534] [Q&A] [answered]
Hi,

I have a question for tegra-flash-init recipe.

RDEPENDS:{PN} include watchdog-keepalive instead of watchdog would occur error in the do_rootfs stage.

```
The following packages have unmet dependencies:
 watchdog-keepalive : Conflicts: watchdog but 5.16-r1 is to be installed
                      Recommends: kernel-module-softdog but it is not installable
E: Unable to correct problems, you have held broken packages.

```
BUT, I am not sure it is a bug or not.

May I use watchdog instead watchdog-keepalive?

BR,
Mark

**Comment by madisongh** [comment, marked as answer]
You could, if you really wanted to, replace that with the full watchdog daemon (and change the script to start it rather than `wd_keepalive`) - just remove `watchdog-keepalive` from the RDEPENDS setting and append `watchdog` there in your bbappend file.... but do you really need the full features of watchdog during the flashing process?

**Reply by bchoineubility** [reply]
I have tested sorted what I need, demo distro would great also, but, I have look into all package.

I am sorry If I disturb you.

BR,
Mark
